src/core/jit/exec.rs

```diff
@@ -70,14 +75,31 @@ where
     async fn init(&mut self) {
         join_all(self.plan.as_nested().iter().map(|field| async {
             let ctx = Context::new(&self.request);
+            let mut arg_map = indexmap::IndexMap::new();
+            for arg in field.args.iter() {
+                let name = arg.name.as_str();
+                let value: Option<Input> = arg
+                    .value
+                    .clone()
+                    // TODO: default value resolution should happen in the InputResolver
+                    .or_else(|| arg.default_value.clone());
+
+                if let Some(value) = value {
+                    arg_map.insert(name, value);
+                } else if !arg.type_of.is_nullable() {
+                    // TODO: throw error here
+                    todo!()
```

Comment:
Can handle error here instead of `todo`

Reply:
not that easy here since the error is generic in exec and we can't actually return any concrete error here.

I think it should be better resolved after changes in error handling for jit